reference/core/ambassador.md

```diff
@@ -91,6 +91,10 @@ spec:
   # envoy_log_path defines the path of log envoy will use. By default this is standard output
   # envoy_log_path: /dev/fd/1
 
+  # envoy_log_format defines the envoy log line format.
+  # see https://www.envoyproxy.io/docs/envoy/v1.8.0/configuration/access_log for a complete list of operators
```

Comment:
This should probably be https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log instead of tying it to Envoy 1.8.0 specifically...

Reply:
Done, thanks.